AFQ/segmentation.py

```diff
@@ -1067,6 +1066,34 @@ def segment_reco(self, tg=None):
         return fiber_groups
 
 
+def sl_curve(sl, n_points):
```

Comment:
Please add unit testing for this and the other new function. 

Also, could you please add a numpy-style docstring?